package.json

```diff
@@ -6,9 +6,11 @@
   "version": "1.3.2",
   "dependencies": {
     "commander": "^2.6.0",
+    "debug": "^2.1.1",
     "denodeify": "^1.2.0",
     "es6-promise": "^2.0.1",
     "haikro": "^1.9.0",
+    "heroku-client": "^1.9.1",
```

Comment:
Also as an alternative haikro already supports provision in this sense through it's create task.  That would need expanding to allow organization to be set though.